### PR TITLE
SendTimestamps: add setting to disable message replacement

### PR DESCRIPTION
**Pull request by jedenastka**
SendTimestamps adds two features allowing to send timestamps: the button in the input area, and the replacement of timestamps in message contents. This allows disabling content replacement if the user only wants to have the input area button.

Something I'm not sure on is whether a corresponding setting for the input area button should be added as well. The description of the setting might also use some improvement.

### CODE REVIEW

File: src/plugins/sendTimestamps/index.tsx [hunk 1]
```diff
@@ -77,7 +86,7 @@ function PickerModal({ rootProps, close }: { rootProps: ModalProps, close(): voi
                 />
 
                 <Forms.FormTitle>Timestamp Format</Forms.FormTitle>
-                <Select
+                <Selec
```
Comment (AutumnVN) [inline src/plugins/sendTimestamps/index.tsx:89]: you ate the letter t

Comment (Vendicated) [reply src/plugins/sendTimestamps/index.tsx:89]: yummy

Comment (jedenastka) [reply src/plugins/sendTimestamps/index.tsx]: My sleepy ass forgot to check `git diff`...

**Comment by jedenastka** [comment]
Forgot to run `pnpm test` after renaming the variable as well.

**Comment by Vendicated** [comment]
thanks!

**Pull request merged**